spanner/api/Spanner.Samples.Tests/SpannerFixture.cs

```diff
@@ -388,6 +433,13 @@ private async Task InitializeBackupAsync()
         }
     }
 
+    private async Task InitializePostgreSqlDatabaseAsync()
```

Comment:
Initialize Database creates a PG database as well as couple of tables namely Singers and Albums. There is another sample which depicts the ordering of NULL (Nulls First, Nulls Last etc.) that creates a different version of Singers table and inserted data is then ordered in different ways. To keep the sample consistent and inline with other languages and to avoid the chances of duplicate data being inserted, I also created a CreateEmptyPostgreSqlDatabase method. I will also share this sample in next batch.

Reply:
OK, so then CreateEmpty... should be on the PR that uses it if posssible.

Reply:
Done.

Reply:
`CreateEmptyPostgreSqlDatabaseAsync` and `ListTableNamesAsync` are now removed from fixture.